node-install

```diff
@@ -28,7 +28,11 @@ if [ "x"$FLAG_HELP != "x" -o "x$LOCATION" = "x" ]; then
 fi
 
 if [ "x$ARCH" = "x" ]; then
-    ARCH="linux-x64"
+    if [ "x$(arch)" = "xaarch64" ]; then
```

Comment:
Is this a mistake? The output of `arch` on my M1 mac is:
```
tagomoris$ arch
arm64
```

Reply:
Linux (native and on `Docker Desktop for Mac` on M1 mac) reports `aarch64`. Please check out `docker info | grep arch` and `docker run --rm debian:buster arch`. This script does not work w/o specifying the `$ARCH` on native mac anyway, and making the default comfortable for any combinations of os and arch will be a tough work.

Reply:
Oh! I missed that this change is only for Linux environment in Docker containers. Never mind.
I also confirmed the output:
```
tagomoris$ docker run -it ubuntu "arch"
aarch64
```